python-avd/pyavd/_eos_designs/structured_config/network_services/vxlan_interface.py

```diff
@@ -305,6 +236,20 @@ def _overlay_her_flood_lists(self: AvdStructuredConfigNetworkServicesProtocol) -
 
         return overlay_her_flood_lists
 
-    @cached_property
-    def _multi_vtep(self: AvdStructuredConfigNetworkServicesProtocol) -> bool:
-        return self.shared_utils.mlag is True and self.shared_utils.evpn_multicast is True
+    def _check_for_duplicate(
```

Comment:
Let's redesign so we only call this once at the end, where it parses self.structured_config looking for duplicate VNIs between vlans (and soon vlan_ranges) and VRFs.